audio/out/ao_lavc.c

```diff
@@ -93,16 +93,6 @@ static void select_format(struct ao *ao, const AVCodec *codec)
     }
 }
 
-static void on_ready(void *ptr)
-{
-    struct ao *ao = ptr;
-    struct priv *ac = ao->priv;
-
-    ac->worst_time_base = encoder_get_mux_timebase_unlocked(ac->enc);
```

Comment:
Where is this set now? Looks like we need to wait for this to become available.

Reply:
This can just be called after the encoder_init.